Client/Frontend/Home/Pocket/FxHomePocketCollectionCell.swift

```diff
@@ -9,14 +9,17 @@ struct FxHomePocketCollectionCellUX {
     static let numberOfItemsInColumn = 3
     static let discoverMoreMaxFontSize: CGFloat = 26 // Title 3 xxxLarge
     static let numberOfItemsInSection = 11
+    static let numberOfSponsoredItemsInSection = 2
+    static let indexOfFirstSponsoredItem = 1
+    static let indexOfSecondSponsoredItem = 9
     static let fractionalWidthiPhonePortrait: CGFloat = 29/30
     static let fractionalWidthiPhoneLanscape: CGFloat = 7/15
 }
 
 class FxHomePocketCollectionCell: UICollectionViewCell, ReusableCell {
 
     // MARK: - Properties
-    var viewModel: FxHomePocketViewModel?
+    var viewModel: FxHomePocketViewModel!
```

Comment:
Does this need to be forced?

Reply:
I tend to use implicitly unwrapped optionals in the context where I can't inject dependencies in the `init` but they are required and I know for sure they've been passed. If not, it's a programmer mistake and it should crash in the development. That helps at the point of usage because we don't have to conditionally unwrap it every time we need to use it. I know the crash part is a bit scary, but if the team feels it's not ok I can revert it to be optional.